src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={classNames(
+              selectedTab.title === tab.title && 'is-active',
+            )}
```

Comment:
firstly don't check by `title`, use `id` for that

secondly please check syntax of using `classnames`

it should be like `className={classNames({ 'is-active': tab.id === selectedTab.id })}`